code/contracts/ONEWallet.sol

```diff
@@ -24,12 +25,12 @@ contract ONEWallet {
         bool completed;
     }
 
-    uint32 constant REVEAL_MAX_DELAY = 60;
+    uint32 constant REVEAL_MAX_DELAY = 60; 
 
     //    bool commitLocked; // not necessary at this time
     Commit[] commits; // self-clean on commit (auto delete commits that are beyond REVEAL_MAX_DELAY), so it's bounded by the number of commits an attacker can spam within REVEAL_MAX_DELAY time in the worst case, which is not too bad.
 
-    uint256[64] ______gap; // reserved space for future upgrade
+    uint256[64] ______gap; // reserved space for future upgrade // IH: I am not sure how this work
 
     constructor(bytes32 root_, uint8 height_, uint8 interval_, uint32 t0_, uint32 lifespan_, uint8 maxOperationsPerInterval_,
```

Comment:
The contract was not payable. This is addressed in https://github.com/polymorpher/one-wallet/commit/4f3f9df87b5f7570feb4af68e461390a0d4a2e64